kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/maps/homework/SchoolDirectory.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.collections.adv.maps.homework;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SchoolDirectory {
+
+    public static void main(String[] args) {
+
+        Map<Principal, School> schools = new HashMap<>();
```

Comment:
Klucz w mapie musi **koniecznie** miec nadpisane equals i hashcode. 
Przypomnij mi prosze, to wrocimy do tego na spotkaniu